2019/f1040.py

```diff
@@ -305,7 +305,7 @@ def child_tax_credit(f, inputs):
         if w['9'] == w['10']:
             raise RuntimeError('TODO: additional child tax credit')
             return None
-        w['11'] = w['9'] - w['10']
+        w['11'] = w['9'] - (w['10'] if w['10'] is not None else 0)
```

Comment:
Good find. Line 305 is a bit concerning to me as well. Instead of doing this, how about appending "or 0" to the end of line 304? That should take care of everything.

Would you mind fixing this for 2018 as well?

Reply:
Good idea, done.